package.json

```diff
@@ -42,7 +42,7 @@
     "replace-placeholders-in-build": "node ./replace.build.js",
     "start": "node dist/apps/api/main",
     "start:client": "ng serve client --configuration=development-en --hmr -o",
-    "start:prod": "node apps/api/main",
+    "start:prod": "yarn database:migrate && yarn database:seed && node main",
```

Comment:
Could you please update the README.md, some steps can be left out I guess.

Reply:
I don't think `database:baseline` is still needed because `database:migrate` will do it. 

I will  complete this PR by removing it and update the README